src/main/java/com/crud/library/controller/CopyOfBookController.java

```diff
@@ -0,0 +1,48 @@
+package com.crud.library.controller;
+
+import com.crud.library.domain.CopyOfTheBook;
+import com.crud.library.domain.CopyOfTheBookDto;
+import com.crud.library.domain.Status;
+import com.crud.library.mapper.CopyOfTheBookMapper;
+import com.crud.library.service.CopyService;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping("/v1/library/copy")
+public class CopyOfBookController {
+
+    private CopyService copyService;
+    private CopyOfTheBookMapper copyOfTheBookMapper;
+
+    public CopyOfBookController(CopyService copyService, CopyOfTheBookMapper copyOfTheBookMapper) {
```

Comment:
nazwałbym to `BookCopyMapper`. nie spotyka się raczej nazw w stylu `CopyOfBook` itp